### ISSUE TITLE
Use Manual Install, Can not use Command Palette

### ISSUE THREAD

**Issue by visoeclipse** [revolunet/sublimetext-markdown-preview#335]
when i download the zip, rename it to Markdown Preview.sublime-package and copy it into the Installed Packages/ directory

then i use command palette can not found: Markdown Preview: Preview in browser

sublime version: 3103

**Comment by visoeclipse** [comment]
no body use manual install ?

**Comment by facelessuser** [comment]
This plugin is best used with Package Control.  Recently Package Control dependencies were added instead of trying to ship the plugin with the dependencies: https://github.com/revolunet/sublimetext-markdown-preview/blob/master/dependencies.json.  In order to ship with the dependencies, we had to modify (sometimes heavily) the module to get them to work from the Markdown Preview folder.  It was a huge pain due to the way some of those dependencies were written (I won't go into details here).

Unfortunately this makes it difficult on those who (for whatever reason) choose to not use Package Control, but from a maintenance standpoint, this makes things much easier on us.
